test/fixtures/full_config.yml

```diff
@@ -2608,14 +2608,14 @@ Style/BitwisePredicate:
 Style/BlockComments:
   Description: Do not use block comments.
   StyleGuide: "#no-block-comments"
-  Enabled: false
+  Enabled: true
   VersionAdded: '0.9'
   VersionChanged: '0.23'
 Style/BlockDelimiters:
   Description: Avoid using {...} for multi-line blocks (multiline chaining is always
     ugly). Prefer {...} over do...end for single-line blocks.
   StyleGuide: "#single-line-blocks"
-  Enabled: false
+  Enabled: true
```

Comment:
I find requiring `do end` for multiline blocks makes chaining look worse:

```
map do |x|
end.filter do |x|
end
```

vs

```
map { |x|
}.filter { |x|
}
```

It looks like there's a `braces_for_chaining` style, but I don't know if that's strictly better

Reply:
Good point. I'll keep this out.